graphql/studentPortal/users/fetchUserRole.ts

```diff
@@ -1,18 +1,21 @@
 import { gql } from "@apollo/client";
 
 export const FETCH_USER_ROLE = gql`
-  query FetchUserRole($_id: String = "") {
-    users(where: { id: { _eq: $_id } }) {
-      userRole {
-        value
-      }
+query FetchUserRole($_id: String = "") {
+  users(where: {id: {_eq: $_id}}) {
+    created_at
+    userRole {
+      value
     }
   }
+}
+
 `;
 
 export type FetchUserRoleData = {
   users: UserRoleData;
 };
 export type UserRoleData = {
-  userRole: "student" | "coach";
+  userRole: "student" | "coach"| "freemium";
+  created_at: Date;
```

Comment:
created_at isn't  really part of UserRoleData, I might rename this class to `UserData`